src/status_im/common/bottom_sheet/view.cljs

```diff
@@ -119,7 +119,7 @@
          [blur/ios-view
           {:style         style/shell-bg
            :blur-radius   (or blur-radius 20)
-           :blur-type     :transparent
+           :blur-type     :dark
```

Comment:
dunno how it's related, because it's only for shell? true ? but in the description not shell screens. cc @Parveshdhull

Reply:
hi @flexsurfer, for onboarding screen we are passing :shell? true
https://github.com/status-im/status-mobile/blob/40e6c44b996af3cb457d58231a2cfc84f6ebd943/src/status_im/contexts/onboarding/new_to_status/view.cljs#L110

Reply:
@alwx I am not sure if using :blur-type as :dark is right solution.

:dark, :light also creates dark overlay, and if you check screenshot added in description. It is blackish and don't match figma.

Reply:
PR            |  Figma
:-------------------------:|:-------------------------:
![](https://github.com/status-im/status-mobile/assets/17097240/342a011f-5ed3-44d5-bcdf-1a1359ddd8fc)  |  ![](https://github.com/status-im/status-mobile/assets/17097240/2216f450-2e6e-4e7b-8ee3-4d46c0fb659a)

Reply:
@Parveshdhull I played with values, nothing else gets even close to the designs except changing the blur amount. Changing background color makes it lighter or darker but doesn't change the blur itself. But you're right about the overlay the `:dark` value creates, it also seems to be a bad option.

I ended up updating the `:blur-amount` to 32 to get this:
<img width="564" alt="Screenshot 2024-03-20 at 11 26 18" src="https://github.com/status-im/status-mobile/assets/911127/0340666d-318b-44b5-a7e0-f54803ff6fb2">

It's still slightly bluish than the design but it's gonna be hard to eliminate the blue tone simply because the button itself is blue and it's getting blurred instead of disappearing. But at least the amount of blur seems to suit better and look closer to the expected result.

<img width="392" alt="Screenshot 2024-03-20 at 11 26 22" src="https://github.com/status-im/status-mobile/assets/911127/b6240440-1d98-4cfa-befd-cacf5682cb2d">

Reply:
> It's still slightly bluish than the design but it's gonna be hard to eliminate the blue tone simply because the button itself is blue and it's getting blurred instead of disappearing.

It should not disappear. I think is a design bug, I checked figma file and it don't have continue button in background.

- Little curious, why screenshot you shared don't have border-radius as screenshots in issue have.
- Please use another on-boarding screen or activity center for finding right value. It seems this screen is not right in figma itself.